catalogs/qhimm.xml

```diff
@@ -2182,5 +2182,47 @@ New Remake avatars for Cid, Vincent and Cait Sith using Rebirth assets.
       </Tags>
     </Mod>
 
+	  ## Curse Ring Challenge
+    <Mod>
+      <ID>2251c41e-0944-4dcb-a362-1ee6ade026da</ID>
+      <Author>Pure_Mind_Games</Author>
+      <Link></Link>
+      <LatestVersion>
+        <Link>iros://GDrive/https$drive.google.com/file/d/1mHgKmHynUN7oMXtO9bUeujnooPjZEH0m/view?usp=drive_link</Link>
```

Comment:
The format of the link is wrong. Can you please fix it? See other mods for examples.